terraform/hcl/applicable/req_sw_terraform_aws_api_gateway_tls_version.tf

```diff
@@ -0,0 +1,4 @@
+resource "aws_api_gateway_domain_name" "vulnerable_example" {
+  domain_name = "dummy"
+  # security_policy is not set
+}
```

Comment:
[comment]: <> (FrogbotReview Iac 1 1 4 2 "vulnerable_example" /terraform/hcl/applicable/req_sw_terraform_aws_api_gateway_tls_version.tf)

### 🛠️ Infrastructure as Code Vulnerability
	
Severity: ![](https://raw.githubusercontent.com/jfrog/frogbot/master/resources/v2/applicableLowSeverity.png)<br>     low

Finding: `security_policy!=TLS_1_2 was detected`

#### 👇 Details

<details>
<summary> <b>Full description</b> </summary>
<br>
Resource `aws_api_gateway_domain_name` should have `security_policy=TLS_1_2`

Vulnerable example -
```
resource "aws_api_gateway_domain_name" "vulnerable_example" {
  # security_policy is not set
}
```

Secure example -
```
resource "aws_api_gateway_domain_name" "secure_example" {
  security_policy = "TLS_1_2"
}
```

</details>

[[🐸 JFrog Frogbot]](https://github.com/jfrog/frogbot#readme)